sql/mysql/Positive-Technologies/MySqlParser.g4

```diff
@@ -45,22 +45,22 @@ sqlStatement
     | administrationStatement | utilityStatement
     ;
 
-emptyStatement_
```

Comment:
Please, do not remove my hard work fixing this grammar to work across platforms. The underscore at the end of a symbol is used to allow the grammar to work with Go.